api/db/setup_local_vars.sh

```diff
@@ -1,3 +1,3 @@
 source setup_vars.sh
-WORKBENCH_PASSWORD=workbench!pwd
-
+export WORKBENCH_PASSWORD=workbench!pwd
```

Comment:
How do you prevent this from overriding env variables that exist in the host environment (like Circle)?